BGL/include/CGAL/boost/graph/named_function_params.h

```diff
@@ -91,6 +92,7 @@ using boost::visitor;
 #define CGAL_add_named_parameter(X, Y, Z)            \
   enum X { Y };
 #include <CGAL/boost/graph/parameters_interface.h>
+#include <CGAL/Polygon_mesh_processing/internal/parameters_interface.h>
```

Comment:
you don't want to introduce a dependy to PMP in BGL

Reply:
When I discussed this with Simon we said that we should have a dedicated package for named parameters.

Reply:
So I should move the `parameters_interface.h` content to the BGL one?

Reply:
You can move them all into the BGL files now that the np classes are the same.

Reply:
@afabri I don't think there is a need for a dedicated package. NP are from BGL so they are at the correct place. For the moment, each package will have its own NP page documentation as it seems non obvious that they'll have the exact same meaning across packages.

Reply:
ok, for putting it into the  BGL package.  It only makes no  sense to make things depend on PMP. We should also do this for Surface_mesh_simplification (I started doinf this in the branch SMS-parallelization-GF).